### PR TITLE
BETA: Register tcm.is-a.dev

### PR DESCRIPTION
**Pull request by TheCatsMo**
Added `tcm.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/tcm.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheCatsMo",
+        "email": "real.p.e.f.a@gmail.com"
+    },
+    "record": {
+        "CNAME": "themooingcat.netlify.app"
```
Comment (Stef-00012) [inline domains/tcm.json:7]: Netlify is not suppored

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/tcm.json (hunk 1 above)
Comment (TheCatsMo) [reply domains/tcm.json:7]: Why?

Comment (TheCatsMo) [reply domains/tcm.json:7]: Also, do you have any recommendations as to what host I can use that is supported?

Comment (Stef-00012) [reply domains/tcm.json:7]: > Why?

It requires a subdomain of the is-a.dev domain, and if you verify yours, no one else would be able to verify theirs
that's why no one can use netlify (same goes for vercel)

Comment (Stef-00012) [reply domains/tcm.json:7]: > Also, do you have any recommendations as to what host I can use that is supported?

if your site is static you can try github pages or cloudflare pages
https://docs.is-a.dev has the guides for some services

**Pull request closed without merging**